2019-12-09 12:26:31,269 manager INFO 1.1.0
2019-12-09 12:26:31,270 manager INFO Copyright (C) 2003-2010
2019-12-09 12:26:31,270 manager INFO   Noriaki Ando
2019-12-09 12:26:31,270 manager INFO   Intelligent Systems Research Institute, AIST
2019-12-09 12:26:31,272 manager INFO Manager starting.
2019-12-09 12:26:31,272 manager INFO Starting local logging.
2019-12-09 12:26:31,278 manager.namingmanager INFO NameServer connection succeeded: corba/localhost
2019-12-09 12:26:33,296 ManagerServant INFO Master manager not found
2019-12-09 12:26:33,296 manager WARNING Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\OpenRTM_aist\Manager.py", line 1900, in initManagerServant
    reffile = file(self._config.getProperty("manager.refstring_path"),'w')
IOError: [Errno 2] No such file or directory: '/var/log/rtcmanager.ref'

2019-12-09 12:26:33,296 LocalServiceAdmin INFO All the local services are enabled.
2019-12-09 12:26:33,312 ec_base INFO setRate(1000000.000000) done
2019-12-09 12:26:33,312 ec_base INFO setRate(1000.000000) done
2019-12-09 12:26:33,312 rtobject INFO 1 execution context was created.
2019-12-09 12:26:33,328 rtobject INFO Initial active configuration set is default.
2019-12-09 12:26:45,223 SdoServiceAdmin WARNING No available SDO service in the factory: 
2019-12-09 12:26:45,223 SdoServiceAdmin ERROR type  already exists.
2019-12-09 12:26:45,226 SdoServiceAdmin WARNING No available SDO service in the factory: IDL:OpenRTM/ComponentObserver:1.0
2019-12-09 12:26:45,226 SdoServiceAdmin ERROR type f36becaf-9a03-43c7-9a3f-e461c881a42c already exists.
